src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/meta/ModifiersProvider.java

```diff
@@ -137,9 +137,11 @@ default boolean isAbstract() {
     }
 
     /**
-     * Checks that this element is concrete and not abstract.
+     * Returns true if a method is with a real implementation, or if a type can
```

Comment:
"if this element is a method with a concrete implementation, or a type that can be instantiated"